source/Outfit.cpp

```diff
@@ -133,6 +133,9 @@ namespace {
 		{"slowing resistance heat", 0.},
 		{"crew equivalent", 0.},
 
+		{"jumpdrive damage", 0.},
+		{"hyperdrive damage", 0.},
```

Comment:
What about the tooltip? Also wouldnt hyperdriving damage be better since it applies during the entire jump sequence?

Reply:
Eh, not sure if I get what you mean, damage is applied once

Reply:
~~phone reviews~~
I still want tooltips tho

Reply:
you got what you want